src/Symfony/Component/Validator/Tests/Constraints/NotIdenticalToValidatorTest.php

```diff
@@ -87,9 +87,6 @@ public function provideInvalidComparisons()
             array($object, '2', $object, '2', __NAMESPACE__.'\ComparisonTest_Class'),
         );
 
-        $immutableDate = new \DateTimeImmutable('2000-01-01');
-        $comparisons[] = array($immutableDate, 'Jan 1, 2000, 12:00 AM', $immutableDate, 'Jan 1, 2000, 12:00 AM', 'DateTime');
```

Comment:
because it's wrong, and already tested thanks to https://github.com/nicolas-grekas/symfony/blob/m-fx/src/Symfony/Component/Validator/Tests/Constraints/AbstractComparisonValidatorTestCase.php#L37